src/python_be.h

```diff
@@ -286,7 +286,8 @@ class ModelInstanceState : public BackendModelInstance {
   std::unique_ptr<IPCMessage> received_message_;
   std::vector<std::future<void>> futures_;
   std::unique_ptr<boost::asio::thread_pool> thread_pool_;
-  std::unordered_map<void*, std::shared_ptr<InferPayload>> infer_payload_;
+  std::unique_ptr<boost::asio::thread_pool> decoupled_response_thread_pool_;
+  std::unordered_map<intptr_t, std::shared_ptr<InferPayload>> infer_payload_;
```

Comment:
I see- we changed to intptr_t for consistency with the other map?

Reply:
Yes, that's correct.